nesta_ds_utils/networks/build.py

```diff
@@ -53,11 +58,51 @@ def build_coocc(
 
     # edge weights are all times a pair of tokens have co-occured in the same sequence
     edge_weights = _cooccurrence_counts(sequences)
+
+    # if using similarity metrics as edge attributes, calculate those
+    if "jaccard_similarity" in edge_attributes:
+        jaccard_similarity = _jaccard_similarity(edge_weights, all_tokens)
+    if "association_strength" in edge_attributes:
+        association_strength = _association_strength(edge_weights, all_tokens)
+    if "cosine" in edge_attributes:
+        cosine_sim = _cosine_sim(edge_weights, all_tokens)
+    if "inclusion_index" in edge_attributes:
```

Comment:
Suggest dropping the suffixes here and making clear whether similarity or distance is used in the docstring.

```suggestion
    if "jaccard" in edge_attributes:
        jaccard_similarity = _jaccard_similarity(edge_weights, all_tokens)
    if "association" in edge_attributes:
        association_strength = _association_strength(edge_weights, all_tokens)
    if "cosine" in edge_attributes:
        cosine_sim = _cosine_sim(edge_weights, all_tokens)
    if "inclusion" in edge_attributes:
```